drivers/timers/Kconfig

```diff
@@ -366,13 +366,21 @@ if WATCHDOG_AUTOMONITOR
 
 choice
 	prompt "Auto-monitor keepalive by"
-	default WATCHDOG_AUTOMONITOR_BY_TIMER
+	default WATCHDOG_AUTOMONITOR_BY_WDOG
 
 config WATCHDOG_AUTOMONITOR_BY_CAPTURE
 	bool "Capture callback"
```

Comment:
Please include a Help description about this symbol, a newcomer user could become confused about it.

Reply:
Done.